.storybook/blocks/ComponentDetails.jsx

```diff
@@ -88,6 +88,44 @@ export const StatusLight = styled.span(({ variant = "positive", ...props }) => `
 	margin-block-end: 1px;
 `);
 
+export const ResourceSection = styled.section`
+	display: flex;
+	flex-flow: row wrap;
+	align-items: center;
+`;
+	
+export const ResourceLink = styled.a`
+	position: relative;
+	display: inline-flex;
+	flex-direction: row;
+	align-items: center;
+	margin-block-end: 16px;
+	margin-inline-end: 16px;
+	box-sizing: border-box;
+	text-decoration: none;
+	min-inline-size: 100px;
+	border: 1px solid transparent;
+	border-radius: 5px;
+	border-color: rgb(230, 230, 230);
+	overflow: hidden;
```

Comment:
I originally wanted to potentially use our custom properties to accommodate light & dark mode, as well as any updates that come with spacing/border/etc. stuff in S2. They were all coming in as undefined, which is why I went with hard coding. 

I'm not sure our Storybook has a light/dark mode switcher quite to the extent of the docs site's (which changes all font colors and background colors everywhere on the page), so is the concern that colors are hardcoded valid currently? Right now, our light/dark mode toggle only applies to the story container and component itself, not the entire page.

Reply:
I think keeping the colors hardcoded here is fine for now, it's nice that the docs site switches color modes but our docs pages aren't doing that right now, we can always change it down the line if that changes.